VERSION

```diff
@@ -1 +1 @@
-0.1.15
+0.1.18
```

Comment:
why are we bumping `15 -> 18`?

Reply:
The intermediate versions are in earlier commits to this branch - just need a discrete version number for each new attempt on PIE.